doc/api/timers.md

```diff
@@ -28,7 +28,7 @@ functions that can be used to control this default behavior.
 added: v9.7.0
 -->
 
-* Returns: {Immediate}
+* Returns: {Immediate} a reference to the `immediate`
```

Comment:
I think this should be either ``` a reference to `immediate` ``` or ``` a reference to the `Immediate` ``` and similarly below.

The first one means: A reference to the variable named "immediate".

The second one means: A reference to the instance of the `Immediate` object.

Reply:
I will fix with the former: we already have type mentioned and the lowercased variant makes it clear that `this` is returned.